willyserge/broadcaster
# #169825089 a user should be able to sign in

#### What does this PR do?
  an endpoint whre a user can enter his email and passoword
#### Description of Task to be completed?
  a working signin endpoint
 /api/v1/auth/signin
#### How should this be manually tested?

-  clone this repo
-  run npm init in cmd
-  run npm install to install all used dependencies
-  run npm npm start  to start server
- check your port number the app is running on
-open postman and go to http://localhost:your port number /api/v1/auth/signup

#### What are the relevant pivotal tracker stories?
#169825089
